build-scripts/postinstall.js

```diff
@@ -62,6 +64,7 @@ shelljs.rm( "-rf",
 	path.join( repoPaths.root, "build-scripts", "generate-functions.js" ),
 	path.join( repoPaths.root, "build-scripts", "helpers", "header-paths.js" ),
 	path.join( repoPaths.installLibraries, "*.a" ),
+	path.join( repoPaths.installLibraries, "*.so" ),
```

Comment:
You cannot remove .so files from `iotivity-installed/lib` because that breaks Linux.

Reply:
I don't understand why Travis didn't run on this PR. I tried running it after adding this commit and Linux was broken because of the missing `liboctbstack.so`.